subsys/bluetooth/controller/Kconfig

```diff
@@ -169,7 +169,7 @@ config BT_CTLR_SDC_SCAN_BUFFER_COUNT
 
 config BT_CTLR_SDC_PERIODIC_SYNC_BUFFER_COUNT
 	int "Number of periodic synchronization receive buffers"
-	default 2
+	default 3
 	range 2 20
 	help
 	  The buffers are used for processing incoming packets
```

Comment:
Can you also document that 3 buffers are needed for chaining?

Reply:
where? in the kconfig help below?